flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1626,41 +1626,29 @@ class FMCMainDisplay extends BaseAirliners {
         return this._routeTripFuelWeight;
     }
 
-    getOrSelectVORsByIdent(ident, callback) {
-        this.dataManager.GetVORsByIdent(ident).then((navaids) => {
-            if (!navaids || navaids.length === 0) {
-                this.addNewMessage(NXSystemMessages.notInDatabase);
+    _getOrSelectWaypoints(getter, ident, callback) {
+        getter(ident).then((waypoints) => {
+            console.log(waypoints);
+            if (!waypoints || waypoints.length === 0) {
```

Comment:
**suggestion**: As `GetVORsByIdent`, `GetNDBsByIdent`, and `GetWaypointsByIdent` always return an `Array` the `length` check will suffice. Consider removing `!waypoints`
```suggestion
            if (waypoints.length === 0) {
```